tree.go

```diff
@@ -405,11 +406,30 @@ type nodeValue struct {
 // made if a handle exists with an extra (without the) trailing slash for the
 // given path.
 func (n *node) getValue(path string, params *Params, unescape bool) (value nodeValue) {
-	var skipped *skip
+	var (
+		skipped *skip
+		nroot   = &(*n)                                // not found `level 1 router` use nroot
+		ldi     = len(strings.Split(path, "/")[1]) + 1 // level 1 router '/' index
+		ri      int                                    //record index
```

Comment:
these names are a little too short and cryptic for my taste. mind making them longer? (i.e. `recordIndex`)

Reply:
thanks for your review, more comments have been added.